src/requisition-full-supply/full-supply.controller.js

```diff
@@ -171,6 +171,20 @@
             return column.name === TEMPLATE_COLUMNS.SKIPPED;
         }
 
+        /**
+         * @ngdoc method
+         * @methodOf requisition-full-supply.controller:FullSupplyController
+         * @name canEditRequisition
+         *
+         * @description
+         * Determines whether the requisition is editable.
+         *
+         * @return {Boolean} true if requisition is editable
+         */
+        function canEditRequisition() {
```

Comment:
I don't know if there is any simple way to do this - do you have any suggestions?